docs/examples/full.rst

```diff
@@ -38,40 +38,40 @@ The purpose of this example is to demonstrate how an actual script, rather than
             )
         )
 
-        async with app.resource(Incident) as r:
-            created = await r.create({
-                Incident.short_description: "Test incident",
-                Incident.description: "This is just a test.",
+        async with app.resource(IncidentSchema) as Incident:
```

Comment:
Yes, I like it. But just *incident* (in lowercase) is better, as this is an object - not a class.

Reply:
fixed.